lib/beaker-answers/answers.rb

```diff
@@ -118,6 +153,24 @@ def answers
     end
 
     # This converts a data hash provided by answers, and returns a Puppet
+    # Enterprise compatible hiera config file ready for use.
+    #
+    # @param [Beaker::Host] host Host object in question to generate the answer
+    #   file for.
+    # @return [String] a string of parseable hocon
+    # @example Generating an answer file for a series of hosts
+    #   hosts.each do |host|
+    #     answers = Beaker::Answers.new("2.0", hosts, "master")
+    #     create_remote_file host, "/mypath/answer", answers.answer_hiera(host)
+    #  end
+    def answer_hiera(host)
+      # Render pretty JSON, because it is a subset of HOCON
+      json = JSON.pretty_generate(answers[host.name])
```

Comment:
The answers returned by hiera_answers is not a hash of host => answer pairs (correctly), but this is trying to look up by host.name.  I think answer_hiera needs a spec?